ios/RNSScreen.mm

```diff
@@ -295,6 +295,14 @@ - (void)addSubview:(UIView *)view
   }
 }
 
+- (void)postNotificationWithEvent:(NSObject<RCTEvent> *)event
+{
+  NSDictionary *userInfo = [NSDictionary dictionaryWithObjectsAndKeys:event, @"event", nil];
+  [[NSNotificationCenter defaultCenter] postNotificationName:@"RCTNotifyEventDispatcherObserversOfEvent_DEPRECATED"
+                                                      object:nil
+                                                    userInfo:userInfo];
+}
```

Comment:
I don't like naming of this method, as this method sends very specific notification & we should either have this reflected in method name (literally or with some other applicable adjective) or take this as a param. 

E.g. `postNotificationForEventDispatcherObserversWithEvent` or smth.

Reply:
Done